src/_help/organizations/personal-user-account-migration.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Personal user account migration
+---
+
+- TOC
+{:toc}
+
```

Comment:
I think we should add a notice about the fact that this page only concerns users with an account created before last week. WDYT?